packages/engine/Source/Core/IndexDatatype.js

```diff
@@ -109,12 +109,12 @@ IndexDatatype.validate = function (indexDatatype) {
 };
 
 /**
- * Creates a typed array that will store indices, using either <code><Uint16Array</code>
- * or <code>Uint32Array</code> depending on the number of vertices.
+ * Creates a typed array that will store indices, using either <code><Uint8Array</code>
+ * or <code><Uint16Array</code> or <code>Uint32Array</code> depending on the number of vertices.
```

Comment:
Minor readability tweak:
```suggestion
 * Creates a typed array that will store indices, using either <code><Uint8Array</code>,
 * <code><Uint16Array</code>, or <code>Uint32Array</code> depending on the number of vertices.
```